gcc/config/riscv/corev.md

```diff
@@ -0,0 +1,264 @@
+;; Machine description for RISC-V MAC operations.
```

Comment:
General comment, after this is merged, you will need to move all the other CORE-V specific machine description code into this file.